SPECS/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,273 @@
+%define modprobe_version 450.57
+%define _major 1
+Summary:        NVIDIA container runtime library
+Name:           libnvidia-container
+Version:        1.3.3
+Release:        2%{?dist}
+License:        BSD-3-Clause AND Apache-2.0 AND GPL-3.0-or-later AND LGPL-3.0-or-later AND MIT AND GPL-2.0-only
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/libnvidia-container
+#Source0:     https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+#Source1:     https://github.com/NVIDIA/nvidia-modprobe/archive/%%{modprobe_version}.tar.gz
+Source1:        nvidia-modprobe-%{modprobe_version}.tar.gz
+Patch0:         common.mk.patch
+Patch1:         libtirpc.patch
+Patch2:         nvidia-modprobe.patch
+BuildRequires:  libseccomp-devel
+BuildRequires:  libtirpc-devel
+BuildRequires:  make
+BuildRequires:  rpcsvc-proto
+BuildRequires:  which
+
+%description
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+%prep
+%setup -q
+%patch0 -p1
+%patch1 -p1
+
+mkdir -p deps/src/nvidia-modprobe-%{modprobe_version}
+tar -C deps/src/nvidia-modprobe-%{modprobe_version} --strip-components=1 -xzf %{SOURCE1}
+%patch2 -p1 -d deps/src/nvidia-modprobe-%{modprobe_version}
+touch deps/src/nvidia-modprobe-%{modprobe_version}/.download_stamp
+
+
+%build
+%make_build WITH_LIBELF=yes
+
+%install
+DESTDIR=%{buildroot} make install prefix=%{_prefix} exec_prefix=%{_prefix} bindir=%{_bindir} libdir=%{_libdir} includedir=%{_includedir} docdir=%{_licensedir} WITH_LIBELF=yes
```

Comment:
Please either use `%make_install` to get rid of the standard arguments we're passing here manually, or write one argument per line.

Reply:
changed format using %make_install was causing issues

Reply:
Can you say what kind of issue and what you've tried? I was thinking something along these lines:

```
DESTDIR=%{buildroot} %make_install WITH_LIBELF=yes
```

Reply:
I did try 
DESTDIR=%{buildroot} %make_install WITH_LIBELF=yes
This resulted in debugfiles.list being empty, it seems like something is not be place correctly when using make_install for this spec file